collect_app/src/main/res/values/strings.xml

```diff
@@ -660,4 +660,5 @@
     <string name="current_predicate_warning_title">Action needed</string>
     <string name="current_predicate_forum">Read details</string>
     <string name="current_predicate_continue">Continue</string>
+    <string name="notification_channel_name">General notifications</string>
```

Comment:
Is this only shown in Settings in the context of configuring the Collect app? If so, this is probably OK for now (we should probably have upload and form update channels at some point?). If not, it might need to include the app name in it.

Reply:
> Is this only shown in Settings in the context of configuring the Collect app?

yes, it is.

> we should probably have upload and form update channels at some point?

I don't think so. if someone, for example, does not want to be notified about for updates then can just disable this option in our settings. i can't see a reason for creating separate channels.

Reply:
Of course, someone might need to disable auto send notifications...

Reply:
It's up to you. I can adde separate channesl if you think it's needed.

Reply:
True for just getting the form update notification -- a user can just turn that off at the app level. I can imagine a user wanting to automatically update forms in the background and NOT get a notification for that. That's not something that can be set at the app level. 

But I really don't think anyone will notice or care so let's not worry about it for now.

Reply:
You're right about auto-send. I still don't think we need to do it now, though.